x-pack/plugins/ml/server/routes/job_service.ts

```diff
@@ -603,7 +609,9 @@ export function jobServiceRoutes({ router, routeGuard }: RouteInitialization) {
           query,
           aggFieldNamePairs,
           splitFieldName,
-          runtimeMappings
+          runtimeMappings,
+
```

Comment:
Same for this extra line here as well

Reply:
https://github.com/elastic/kibana/pull/91830/commits/8ca21bb9de9c182d0c95e40cd550f4596af8f8e0